### ISSUE TITLE
Hard-coded NAAN in biocode-fims.js

### ISSUE THREAD

**Issue by GoogleCodeExporter** [biocodellc/biocode-fims#85]
```
The very first line of biocode-fims.js contains a hard-coded NAAN value:

// Must set global variable naan here to check a spreadsheet's naan
var naan = 21547

This should NOT be here... we need to fetch the NAAN from the defined NAAN in 
the BCID settings properties file when the fims page is loaded (try to do this 
async).  

```

Original issue reported on code.google.com by `jdec...@gmail.com` on 30 Jan 2015 at 12:03

**Comment by jdeck88** [comment]
NAAN is specified in FIMS properties file so this should be a service call, like:

{FIMS_ROOT}/getNAAN

will require Javascript updates in web_nmnh and web respectively...

**Comment by rodney757** [comment]
implemented in commit d5cf7cc8f1fbfea28ae528e2e1e90d3a6d01ea12
